Servers/UI/OJS.Servers.Ui/ClientApp/src/components/guidelines/forms/FormControl.tsx

```diff
@@ -24,7 +25,7 @@ interface ILabelInternalProps extends IHaveChildrenProps {
     forKey: string,
 }
 
-const LabelInternal = ({ text, className, forKey, children }: ILabelInternalProps) => {
+const LabelInternal = ({ id, text, className, forKey, children }: ILabelInternalProps) => {
```

Comment:
This is not enough as `LabelInternal` is internal, i.e. private to this file.

You should add `labelId` in `FormControl`'s props as well.